sdk/spring/scripts/compatibility_insert_dependencymanagement.py

```diff
@@ -148,20 +142,20 @@ def get_dependency_management_content():
 """
 
 
-def get_properties_contend_with_tag(spring_cloud_version):
+def get_properties_contend_with_tag():
     return """
   <properties>
     {}
   </properties>
   
-  """.format(get_properties_contend(spring_cloud_version))
+  """.format(get_properties_contend())
 
 
-def get_properties_contend(spring_cloud_version):
+def get_properties_contend():
```

Comment:
better pass the versions from the method's parameters, instead of getting the args again here, and we should do it for the other methods too.